src/app/cluster/details/external-cluster/external-machine-deployment-list/component.ts

```diff
@@ -45,7 +50,17 @@ export class ExternalMachineDeploymentListComponent implements OnInit, OnChanges
   @Output() machineDeploymentChange$ = new EventEmitter<ExternalMachineDeployment>();
   @ViewChild(MatPaginator, {static: true}) paginator: MatPaginator;
   dataSource = new MatTableDataSource<ExternalMachineDeployment>();
-  displayedColumns: string[] = ['status', 'name', 'labels', 'replicas', 'version', 'os', 'created', 'actions'];
+  displayedColumns: string[] = [
```

Comment:
enum Column {
 // define ENUM for controls
}

I know this is done multiple places but it should be done like e.g 
- PresetListComponent
- MeteringScheduleConfigComponent

Advantanges:
- less chances of error (ts and html) same column name is used e.g [matColumnDef]="column.<NAME>"
- same enum can be used inside template